app/controllers/activity_forms_controller.rb

```diff
@@ -160,6 +160,9 @@ def assign_default_collaboration_type_value_if_nil
   def page_title(step)
     if step == :identifier && @activity.programme?
       t("page_title.activity_form.show.identifier_level_b")
+    elsif step == :commitment
+      optional = @activity.third_party_project? ? "" : " (optional)"
+      t("page_title.activity_form.show.commitment", optional: optional)
```

Comment:
Maybe this all needs to go into the last commit, or the test could be brought over here?